src/main/java/org/traccar/web/client/view/NavView.java

```diff
@@ -151,33 +140,29 @@ public NavView(SettingsHandler settingsHandler,
         uiBinder.createAndBindUi(this);
 
         User user = ApplicationContext.getInstance().getUser();
-        boolean readOnly = user.getReadOnly();
-        boolean admin = user.getAdmin();
-        boolean manager = user.getManager();
 
-        settingsButton.setVisible(admin || manager);
-        settingsAccount.setVisible(!readOnly);
-        settingsPreferences.setVisible(!readOnly);
+        settingsButton.setVisible(user.hasPermission(UserPermission.SERVER_MANAGEMENT)
```

Comment:
Dlaczego jest tutaj taka alternatywa?
Dlaczego USER_MANAGEMENT wystarczy, żeby wejść w ustawienia?

Reply:
``` settingsButton ``` to jest rozwijane menu, które zawiera 3 opcje: Użytkownicy, Globalne i Powiadomienia. Jeśli użytkownik ma prawo ```USER_MANAGEMENT``` to w ustawieniach widzi tylko użytkowników. Jeśli ma tylko ```SERVER_MANAGEMENT```, widzi jedynie Globalne i Powiadomienia.

Reply:
Ok, dzięki